src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```
@@ -148,6 +148,8 @@ err.cannot.read.module.info=cannot read module descriptor from {0}
 err.not.modular.format=selected module {0} ({1}) not in jmod or modular JAR format
 err.signing=signed modular JAR {0} is currently not supported,\
 \ use --ignore-signing-information to suppress error
+err.cannot.determine.target.platform=cannot determine target platform for image generation
```

```suggestion
err.cannot.determine.target.platform=cannot determine target platform from {0}
```

`{0}` can be the path to `java.base.jmod`  (i.e. `ModuleReference::location`)

---

Done. Additionally, if the `ModuleReference::location` is unknown (it's an `Optional`) then we will print `cannot determine target platform from java.base module`